examples/stack_breakdown.py

```diff
@@ -1,22 +1,26 @@
-from pybamm_tea import TEA
 import pybamm
+from pybamm_tea import tea
```

Comment:
does this work? you shouldn't be able to import `tea` (it's not a method anywhere).